### DISCUSSION TITLE
Can I configure the commit message that is generated by the npx commit-and-tag-version?

### DISCUSSION THREAD

**Discussion by kaspar-naaber** [absolute-version/commit-and-tag-version#134] [Q&A] [answered]
I would like to append [skip ci] to the commit message to prevent running Bitbucket pipelin on the version number and changelog chages. Is it somehow possible?

**Comment by TimothyJones** [comment, marked as answer]
Yes, with `--releaseCommitMessageFormat` or `--message`

**Reply by kaspar-naaber** [reply]
Thanks!
